src/test/librados/tier_cxx.cc

```diff
@@ -189,6 +190,7 @@ void is_intended_refcount_state(librados::IoCtx& src_ioctx,
     }
     break;
   }
+  ASSERT_TRUE(tries < 30);
```

Comment:
Can this actually fail?  Why does ```tries``` need to be outside of the loop?

Reply:
I added this line for the purpose of debugging. The two issues above generated error logs like `ASSERT_TRUE(src_refcount == expected_refcount);`. So, It is hard to catch the cause because I need to read all logs generated from the cluster. This line might help us to recognize the error caused by the retry.